docs/project/commit_access.md

```diff
@@ -0,0 +1,64 @@
+# Commit access
+
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+<!-- toc -->
+
+## Table of contents
+
+-   [Overview](#overview)
+-   [Getting access](#getting-access)
+-   [Removing access](#removing-access)
+
+<!-- tocstop -->
+
+## Overview
+
+Commit access is what allows developers to merge PRs into the main
+[carbon-lang repository](https://github.com/carbon-language/carbon-lang/). It
+also gives the ability to [review, approve, and merge changes](code_review.md).
+
+Developers with commit access are not required to do reviews. It is granted as a
+convenience in order to allow fixing PRs after approval, for example because the
+review noted a minor typo, or because there is a conflict that the PR author
+needs to resolve before merging.
+
+Developers with commit access are expected to make reasonable choices about when
+to review and approve PRs for others, and what changes to make to their own PRs
+without asking for another round of review.
```

Comment:
Seems nice to give some guidance on what developers should be thinking about here:

```suggestion
Developers with commit access are expected to make reasonable choices about when
to approve or merge PRs for others -- generally, the change either needs to be trivially understood without context on the code in question (like a cleanup, typo fix, etc), or the developer should have reasonable context on the code in question. If in doubt, feel free to review but leave approving or merging for someone else.

Similarly, developers with commit access are expected to make reasonable choices about what changes to make to their own PRs
without asking for another round of review prior to merge, even after approval. And again, if in doubt, ask for review.
```

Reply:
Before:

> Developers with commit access are expected to make reasonable choices about when
> to review and approve PRs for others, and what changes to make to their own PRs
> without asking for another round of review.

After:

> Developers with commit access are expected to make reasonable choices about when
> to approve or merge PRs for others -- generally, the change either needs to be trivially understood without context on the code in question (like a cleanup, typo fix, etc),

Note in this section, I think you changed the focus from "when do you need review for changes to your own PRs" to "when do other PRs" need review. I'm trying to adjust phrasing.

Reply:
Nevermind. I think I'm just confused by the emphasis here.